### PR TITLE
Issue/select

### PR DESCRIPTION
**Pull request by BelohvostovAlex**

### CODE REVIEW

File: src/components/Todo/Todo.tsx
Comment (igorTT) [inline src/components/Todo/Todo.tsx:49]: There is logic from another ticket. Wait until it'll be merged or delete it

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/Todo.tsx
Comment (BelohvostovAlex) [reply src/components/Todo/Todo.tsx]: deleted

File: src/components/Todo/todo.scss
Comment (igorTT) [inline src/components/Todo/todo.scss:34]: Why do we need the max-width?

File: src/components/Todo/todo.scss
Comment (igorTT) [inline src/components/Todo/todo.scss:40]: I don't see the rounded borders on the mockup

**Review by igorTT — CHANGES_REQUESTED** [review]
image
Looks like a problem

File: src/components/Todo/todo.scss
Comment (BelohvostovAlex) [reply src/components/Todo/todo.scss:34]: deleted

File: src/components/Todo/todo.scss
Comment (BelohvostovAlex) [reply src/components/Todo/todo.scss:40]: deleted

File: src/components/Todo/Todo.tsx
Comment (igorTT) [inline src/components/Todo/Todo.tsx:36]: Move it to the separate component

File: src/components/Todo/Todo.tsx
Comment (igorTT) [inline src/components/Todo/Todo.tsx:37]: Useless wrapper

File: src/components/Todo/todo.scss
Comment (igorTT) [inline src/components/Todo/todo.scss:37]: Useless padding

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/Todo.tsx
Comment (BelohvostovAlex) [reply src/components/Todo/Todo.tsx:37]: removed

File: src/components/Todo/todo.scss
Comment (BelohvostovAlex) [reply src/components/Todo/todo.scss:37]: removed

File: src/components/Todo/Todo.tsx
Comment (BelohvostovAlex) [reply src/components/Todo/Todo.tsx:36]: done

File: src/components/Todo/TodoContainer.tsx
Comment (igorTT) [inline src/components/Todo/TodoContainer.tsx:17]: What does this state do?
Config eslint on your code editor. It should highlight this kind of errors

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/TodoContainer.tsx
Comment (BelohvostovAlex) [reply src/components/Todo/TodoContainer.tsx:17]: removed state,
yes, i had a warning, thought warnings = okay, no problem, just warning)

File: src/components/Todo/TodoContainer.tsx
Comment (igorTT) [inline src/components/Todo/TodoContainer.tsx:13]: Store this array out of the component. It'll help with memoization.

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/TodoContainer.tsx
Comment (BelohvostovAlex) [reply src/components/Todo/TodoContainer.tsx]: Should i store it outside the func body?

**Review by igorTT — APPROVED** [review]

**Comment by igorTT** [comment]
@BelohvostovAlex , wait for https://github.com/BelohvostovAlex/todo/tree/issue/base will be merged

**Review by igorTT — APPROVED** [review]

**Pull request merged**
